docs/source/daml/intro/daml/daml-intro-6/Parties.daml

```diff
@@ -66,6 +68,7 @@ template Iou
 -- IOU_END
 
 -- PROPOSE_TRANSFER_BEGIN
+      -- This is a choice on the Iou template, with controller owner.
```

Comment:
IMO, say that `owner` is the controller hints at an underlying problem. We should move this example to the new choice syntax.

Reply:
That means introducing the new syntax in the explanation, which I'm not sure we want to do at this point. I'll let @bame-da or @nemanja-da chime in.